src/StatusQ/Components/StatusToastMessage.qml

```diff
@@ -0,0 +1,166 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import QtGraphicalEffects 1.13
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
+Control {
+    id: root
+    width: 343
+    height: !!secondaryText ? 68 : 48
+    anchors.right: parent.right
+
+    property bool open: false
+    property string primaryText: ""
+    property string secondaryText: ""
+    property bool loading: false
+    property string iconName: ""
+    property bool success: false
```

Comment:
I'd expect that there's toast messages that are neither "success" nor "loading" but probably "failure".

How about we rather introduce an enum type, similar to many other components in StatusQ that can have these different states?

Reply:
Same as with the icon settings: We can add it later, but it would require us to change the API so this will be a breaking change.

> I thought about this, but since is nothing specified yet I don't know whether it will be only 2 types or more

We already have "failure" toast messages in Status Desktop today (for failing transactions). The new design just seems to be incomplete on that regard. I guess we want to preserve existing functionality.